[2023-02-10 12:15:37,110] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:15:37,112] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:15:37,112] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:15:37,112] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-10 12:15:37,112] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:15:37,116] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-02-09 00:00:00+00:00
[2023-02-10 12:15:37,117] {standard_task_runner.py:52} INFO - Started process 16219 to run task
[2023-02-10 12:15:37,119] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2023-02-09T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmphwzo_74n', '--error-file', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmpbcxxv9dr']
[2023-02-10 12:15:37,120] {standard_task_runner.py:80} INFO - Job 18: Subtask cria_pasta
[2023-02-10 12:15:37,143] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [running]> on host airdefrancilene.lan
[2023-02-10 12:15:37,159] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-09T00:00:00+00:00
[2023-02-10 12:15:37,160] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T
[2023-02-10 12:15:37,160] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p  "/airflow-alura/pasta" ']
[2023-02-10 12:15:37,164] {subprocess.py:85} INFO - Output:
[2023-02-10 12:15:37,170] {subprocess.py:92} INFO - mkdir: /airflow-alura: Read-only file system
[2023-02-10 12:15:37,171] {subprocess.py:96} INFO - Command exited with return code 1
[2023-02-10 12:15:37,175] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/francilenesilva/Documents/data-engineer/apache-airflow/orquestrando-pipeline1/data-pipeline/airflow-alura/venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 194, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-02-10 12:15:37,176] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20230209T000000, start_date=20230210T121537, end_date=20230210T121537
[2023-02-10 12:15:37,180] {standard_task_runner.py:92} ERROR - Failed to execute job 18 for task cria_pasta (Bash command failed. The command returned a non-zero exit code 1.; 16219)
[2023-02-10 12:15:37,225] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-02-10 12:15:37,233] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-02-10 12:23:22,154] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:23:22,156] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:23:22,156] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:23:22,156] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-10 12:23:22,156] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:23:22,160] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-02-09 00:00:00+00:00
[2023-02-10 12:23:22,161] {standard_task_runner.py:52} INFO - Started process 17378 to run task
[2023-02-10 12:23:22,164] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2023-02-09T00:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmpr0q290ah', '--error-file', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmpq8g9qlq7']
[2023-02-10 12:23:22,165] {standard_task_runner.py:80} INFO - Job 25: Subtask cria_pasta
[2023-02-10 12:23:22,178] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [running]> on host airdefrancilene.lan
[2023-02-10 12:23:22,194] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-09T00:00:00+00:00
[2023-02-10 12:23:22,194] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T
[2023-02-10 12:23:22,195] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p  "/Users/francilenesilva/Documents/data-engineer/apache-airflow/orquestrando-pipeline1/data-pipeline/airflow-alura" ']
[2023-02-10 12:23:22,199] {subprocess.py:85} INFO - Output:
[2023-02-10 12:23:22,204] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-10 12:23:22,210] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20230209T000000, start_date=20230210T122322, end_date=20230210T122322
[2023-02-10 12:23:22,223] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10 12:23:22,230] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-02-10 12:25:25,212] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:25:25,214] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 12:25:25,214] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:25:25,214] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-10 12:25:25,214] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 12:25:25,218] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-02-09 00:00:00+00:00
[2023-02-10 12:25:25,219] {standard_task_runner.py:52} INFO - Started process 17741 to run task
[2023-02-10 12:25:25,221] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2023-02-09T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmpdvnma3sn', '--error-file', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmp6j8ol8h1']
[2023-02-10 12:25:25,222] {standard_task_runner.py:80} INFO - Job 27: Subtask cria_pasta
[2023-02-10 12:25:25,237] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [running]> on host airdefrancilene.lan
[2023-02-10 12:25:25,253] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-09T00:00:00+00:00
[2023-02-10 12:25:25,253] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T
[2023-02-10 12:25:25,254] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p  "/Users/francilenesilva/Documents/data-engineer/apache-airflow/orquestrando-pipeline1/data-pipeline/airflow-alura/pasta" ']
[2023-02-10 12:25:25,257] {subprocess.py:85} INFO - Output:
[2023-02-10 12:25:25,261] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-10 12:25:25,268] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20230209T000000, start_date=20230210T122525, end_date=20230210T122525
[2023-02-10 12:25:25,281] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10 12:25:25,287] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-02-10 13:23:47,273] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 13:23:47,275] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-02-10 13:23:47,275] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 13:23:47,275] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-02-10 13:23:47,275] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-10 13:23:47,279] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2023-02-09 00:00:00+00:00
[2023-02-10 13:23:47,281] {standard_task_runner.py:52} INFO - Started process 24784 to run task
[2023-02-10 13:23:47,283] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'meu_primeiro_dag', 'cria_pasta', 'scheduled__2023-02-09T00:00:00+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/meu_primeiro_dag.py', '--cfg-path', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmp9gquuc8y', '--error-file', '/var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T/tmpam0sgybd']
[2023-02-10 13:23:47,284] {standard_task_runner.py:80} INFO - Job 41: Subtask cria_pasta
[2023-02-10 13:23:47,298] {task_command.py:370} INFO - Running <TaskInstance: meu_primeiro_dag.cria_pasta scheduled__2023-02-09T00:00:00+00:00 [running]> on host airdefrancilene.lan
[2023-02-10 13:23:47,315] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=meu_primeiro_dag
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2023-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-09T00:00:00+00:00
[2023-02-10 13:23:47,315] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/b_/44w2czd50d77jsslrsmc4_sc0000gn/T
[2023-02-10 13:23:47,316] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p  "/Users/francilenesilva/Documents/data-engineer/apache-airflow/orquestrando-pipeline1/data-pipeline/airflow-alura/pasta=2023-02-10T00:00:00+00:00" ']
[2023-02-10 13:23:47,319] {subprocess.py:85} INFO - Output:
[2023-02-10 13:23:47,322] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-10 13:23:47,329] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=meu_primeiro_dag, task_id=cria_pasta, execution_date=20230209T000000, start_date=20230210T132347, end_date=20230210T132347
[2023-02-10 13:23:47,344] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-10 13:23:47,353] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
